lib/webpacker/configuration.rb

```diff
@@ -0,0 +1,55 @@
+# Loads the package.json file from app root to read the webpacker configuration
+
+class Webpacker::Configuration
+  class NotFoundError < StandardError; end
+  class_attribute :instance
+  attr_accessor :config
+
+  class << self
+    def load(path = Rails.root.join("package.json"))
+      self.instance = new(path)
+    end
+
+    def config
+      load if Rails.env.development?
+      instance.config
+    end
+
+    def scripts
+      config[:scripts]
+    end
+
+    def webpacker
+      begin
+        config[:webpacker]
+      rescue NoMethodError
+        raise NotFoundError, "Error: Webpacker core config not found in package.json. Make sure webpacker:install is run successfully"
+      end
+    end
```

Comment:
Instead of having these specific lookups, seems like we simply need something that ensures the lookup is available. Have that problem elsewhere too. Maybe just `lookup!` if we also need a `lookup` that doesn't require that the key is present.